coralproject/talk
# Fixes comment number change when clicking the "Settings" tab.

## What does this PR do?

Fixes an issue where the number of comments listed changes when the user clicks the "Settings" tab. (Bug: https://www.pivotaltracker.com/n/projects/1863625/stories/137367715)

## How do I test this PR?

1. Make some comments.
2. Log out and log in as another user, make some more (there should be comments by multiple users in the stream.
3. Click the "Settings" tab. The number of comments listed should not change.
4. Click the "Comments" tab. All comments, including those not by the currently logged in user, should be present.
